requirements.txt

```diff
@@ -5,9 +5,7 @@ flake8-import-order
 funcsigs==1.0.2
 mock==2.0.0
 pylint
-# pytest is fixed at 3.6.4 because never version (3.7.*) requires scandir, which depends on gcc
-pytest==3.6.4
-pytest-ordering==0.5
+pytest==4.6.11
```

Comment:
What is the reasoning for this change? The change is not documented anywhere and regarding the previous comment, I expect the 3.6.4 version should be kept. If there is not real reasoning for this, the old pytest should be kept.

Reply:
There is a reason. But it was hard to find:) In pytest 3.6 the caplog implementation doesn't have attribute messages (see https://travis-ci.org/github/oamg/leapp-repository/jobs/705807738#L4052). So I bumped the pytest to a higher version. Moreover, we are no longer restricted by the gcc, since we added it to the dependencies

Reply:
OK. So please document it. Nor the commit, nor the file have a comment about the change to make it clear for the others.